DuckDuckGo/Tab/Model/Tab.swift

```diff
@@ -1278,10 +1278,16 @@ extension Tab/*: NavigationResponder*/ { // to be moved to Tab+Navigation.swift
             NSLocalizedDescriptionKey: UserText.webProcessCrashPageMessage
         ])
 
-        if case.url(let url, _, _) = content {
-            self.error = error
+        let isInternalUser = NSApp.delegateTyped.internalUserDecider.isInternalUser
```

Comment:
Tab contains an optional internalUserDecider property, so you could use `internalUserDecider?.isInternalUser == true`

Reply:
Good call! Fixed it.